README.rst

```diff
@@ -98,12 +98,15 @@ Quick Example
 
     bot = commands.Bot(command_prefix='$')
 
-    @bot.command()
-    async def ping(ctx):
-        await ctx.reply('Pong!')
 
-    bot.run('token')
+    @bot.slash_command(description="Replies with pong!")
+    async def ping(interaction: nextcord.Interaction):
+        await interaction.send("Pong!", ephemeral=True)
 
+    bot.run("token")
+
+
+It is recommended that you use ``commands.Bot``, as although both ``nextcord.Client`` and ``commands.Bot`` allow you to interface with Discord, ``commands.Bot`` provides an easier way to make prefix commands with a framework and it is recommended to use ``commands.Bot`` for prefix commands & slash commands. Make sure to not use both versions, though, as only one of them will actually be connected to Discord.
```

Comment:
The wording is unnecessarily long here.

In my opinion, it is not necessary at all to specify where it is recommended since commands.Bot currently allows features that are useful even if you are not using prefix commands.

```suggestion
It is recommended that you use ``commands.Bot``, as although both ``nextcord.Client`` and ``commands.Bot`` allow you to interface with Discord, ``commands.Bot`` provides an easier way to make prefix commands with a framework. Make sure to not use both versions, though, as only one of them will actually be connected to Discord.
```

With @TAG-Epic's suggestion taken into account, it could be simplified to:

```suggestion
It is recommended that you use ``commands.Bot`` if you wish to use prefix commands, as although both ``nextcord.Client`` and ``commands.Bot`` allow you to interface with Discord, ``commands.Bot`` provides an easier way to make prefix commands with a framework. Make sure to not use both versions, though, as only one of them will actually be connected to Discord.
```

@TAG-Epic's opinion is requested on this before applying one of the above changes.

Reply:
Id add a bit of info about cogs & multiple listeners aswell with ext.commands, but other than that it looks very good!